.github/workflows/upload.yml

```diff
@@ -23,6 +19,8 @@ jobs:
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.inputs.tag-version }}
```

Comment:
```suggestion
          ref: ${{ inputs.tag-version }}
```
**suggestion (non-blocking)**: According to [the documentation](https://docs.github.com/en/actions/learn-github-actions/contexts#example-usage-of-the-inputs-context-in-a-manually-triggered-workflow), `inputs` are already available.